client/asset/eth/nodeclient_harness_test.go

```diff
@@ -682,8 +689,12 @@ func testInitiate(t *testing.T) {
 			t.Fatalf("%s: balance error: %v", test.name, err)
 		}
 
+		gasPrice, err := feesAtBlk(ctx, ethClient, receipt.BlockNumber.Int64()-1)
```

Comment:
Does this mean that the transaction was not being included in the block we expected?

Reply:
I guess it just means that the base fee for the a block is encoded in (or calculated from) the previous block.

Reply:
Oh, it's in the header, the latest commit makes more sense for getting the base fee from mined blocks.